scripts/inconsistentVersionsInFSharpScripts.fsx

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env -S dotnet fsi
+
+open System.IO
+
+#load "../src/FileConventions/Library.fs"
+
+let rootDir = Path.Combine(__SOURCE_DIRECTORY__, "../scripts") |> DirectoryInfo
```

Comment:
@realmarv this shouldn't be needed, since the script should only look inside .fsx files anyway

Reply:
(so, rootDir is enough, not scripts/ subfolder)

Reply:
If I ignore DummyFiles in the DetectInconsistentVersionsInFsharpScripts function, the tests will fail because the function is ignoring test-related files. If I don't ignore it, the CI run will fail because the function will find inconsistent versions in test-related dummy files. What do you think I should do?

Reply:
I'm surprised you ask me this question because you have faced this problem with other PRs, and you fixed it by ignoring DummyFiles in the script, not in FileConventions proj

Reply:
Ok but this way `DetectInconsistentVersionsInFsharpScripts` that receives a directory as input will become useless

Reply:
why? I don't understand

Reply:
Because `DetectInconsistentVersionsInFsharpScripts` is the function that finds all .fsx files and calls the `DetectInconsistentVersionsInNugetRefsInFsharpScripts` function on them. If I want to ignore the DummyFiles in the script, I need to call `DetectInconsistentVersionsInNugetRefsInFsharpScripts`  directly in the script